apluslms_roman/backends/__init__.py

```diff
@@ -1,19 +1,21 @@
+import logging
 from collections import namedtuple
 from collections.abc import Mapping
 
 from ..observer import BuildObserver
-
+from ..utils.path_mapping import get_host_path
 
 BACKENDS = {
     'docker': 'apluslms_roman.backends.docker.DockerBackend',
 }
 
-
```

Comment:
This is like a class definition, and thus should be surrounded by 2 new lines. Thus, don't remove this line.